inputs-optional.tf

```diff
@@ -67,6 +67,11 @@ variable "enable_synapse_spark_pool" {
   default     = false
 }
 
+variable "vm_availabilty_zones" {
+  description = "Availability zones for the VMs"
+  default     = ""
+}
+
```

Comment:
Do we need this? doesn't look like its being used, can this be removed please?

Reply:
When I removed it, complaining about the availability zone, that's why I put it back."